packages/core/scripts/voting-upgrade-umip/1_Propose.js

```diff
@@ -11,7 +11,7 @@ const Registry = artifacts.require("Registry");
 const Voting = artifacts.require("Voting");
 const VotingToken = artifacts.require("VotingToken");
 const Governor = artifacts.require("Governor");
-const Umip15Upgrader = artifacts.require("Umip15Upgrader");
+const VotingUpgrader = artifacts.require("VotingUpgrader");
```

Comment:
here we are using `artifacts.require`. however, in PR https://github.com/UMAprotocol/protocol/pull/2444/files we are using `getTruffleContract`. should we consider changing this to use the same syntax everywhere?

Reply:
Good point. Fixed.